amxmodx/amx.cpp

```diff
@@ -4173,3 +4173,43 @@ int AMXAPI amx_GetStringOld(char *dest,const cell *source,int use_wchar)
   dest[len]='\0';     /* store terminator */
   return AMX_ERR_NONE;
 }
+
+
+#include <chrono>
```

Comment:
Is it allowed to use STL instead of AMTL in amxmodx?

Reply:
I don't know

Reply:
I can use another clock?

Reply:
@WPMGPRoSToTeMa maybe you know another better way?

Reply:
@WPMGPRoSToTeMa yes possible in future need add clocks to AMTL and then use it in AMXMODX. I did not attach importance to your message until I ran with this issue: https://github.com/alliedmodders/amxmodx/pull/1026